source/grid/tria.cc

```diff
@@ -1689,7 +1689,7 @@ namespace internal
               default:
                 // in 1d, a node must have one or two adjacent lines
                 if (spacedim==1)
-                  AssertThrow (false, ExcInternalError())
+                  Assert (false, ExcInternalError())
```

Comment:
And here -- this is a safety check for the data we read in. I believe that I could easily construct an input file that would trigger that.